docker/release/nanoserver/Dockerfile

```diff
@@ -46,3 +46,5 @@ RUN Start-Transcript -path C:\Dockerfile.log -append -IncludeInvocationHeader ;
 
 # Persist %PSCORE% ENV variable for user convenience
 ENV PSCORE='"C:\Program Files\PowerShell\Core\PowerShell.exe"'
+
+ENTRYPOINT ["C:\\Program Files\\PowerShell\\Core\\PowerShell.exe"]
```

Comment:
Since it's just a symlink, why not just put it in `C:\Program Files\PowerShell`?